river/drift/eddm.py

```diff
@@ -18,19 +18,28 @@ class EDDM(DriftDetector):
 
     The algorithm works similarly to the DDM algorithm, by keeping
     track of statistics only. It works with the running average
-    distance ($p_i^'$) and the running standard deviation ($s_i^'$), as
-    well as $p^'_{max}$ and $s^'_{max}$, which are the values of $p_i^'$
-    and $s_i^'$ when $(p_i^' + 2 * s_i^')$ reaches its maximum.
+    distance ($p_i'$) and the running standard deviation ($s_i'$), as
+    well as $p'_{max}$ and $s'_{max}$, which are the values of $p_i'$
+    and $s_i'$ when $(p_i' + 2 * s_i')$ reaches its maximum.
 
     Like DDM, there are two threshold values that define the
     borderline between no change, warning zone, and drift detected.
     These are as follows:
 
-    * if $(p_i^' + 2 * s_i^')/(p^'_{max} + 2 * s^'_{max}) < \alpha$ -> Warning zone
-    * if $(p_i^' + 2 * s_i^')/(p^'_{max} + 2 * s^'_{max}) < \beta$ -> Change detected
+    * if $(p_i' + 2 * s_i')/(p'_{max} + 2 * s'_{max}) < \alpha$ -> Warning zone
+
+    * if $(p_i' + 2 * s_i')/(p'_{max} + 2 * s'_{max}) < \beta$ -> Change detected
 
     $\alpha$ and $\beta$ are set to 0.95 and 0.9, respectively.
 
+    **Input:** EDDM expects `value` to be a binary value as input, indicating
+     whether the last sample analyzed was correctly classified.
+
+    - 0: Correclty classifies
```

Comment:
There is a typo in "Correctly"

Reply:
Reworded this comment for clarity in 27ef120bd2b0d3c4e1560e223db91ffba0415b16

Same applies to `DDM`, `EDDM`, `HDDM_A`, and `HDDM_W`.